framework/packages/abstract-std/src/objects/validation/verifiers.rs

```diff
@@ -28,6 +30,8 @@ pub fn validate_link(link: Option<&str>) -> Result<(), ValidationError> {
             Err(ValidationError::LinkInvalidFormat {})
         } else if contains_dangerous_characters(link) {
             Err(ValidationError::LinkContainsDangerousCharacters {})
+        } else if let Err(e) = Url::parse(link) {
+            Err(ValidationError::LinkInvalidUrl(e))
```

Comment:
The reason we didn't do this Url verification from the original audit results last year was because of wasm size increase with minimal benefit. Would you mind checking the size differences before and after?

Reply:
The increase is ~ 160K, this is huge !
We might not want to merge that PR